src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
The reason this was passed as an argument before was to re-use the single declaration from here - https://github.com/numo-labs/isearch-ui/blob/master/lib/tags/index.js#L5-L14

By having the tag metadata in the reducer as well we're maintaining the same data in two different places.